### DISCUSSION TITLE
How does bing ai jailbreak mode work?

### DISCUSSION THREAD

**Discussion by Cl00m** [waylaidwanderer/node-chatgpt-api#273] [Help]
I am a complete newbie, I am so curious about how the jailbreak mode of bing ai works, may someone explain it in a easy to understand way？
Or if  you can give me some guidance about what book or blog or text to read in order to understand it, I would be so grateful cause I am all by myself basically. 
Thanks a lot!

**Comment by djsnipa1** [comment]
I would also like a quick and dirty answer as to how it works (command to run, etc. with just a brief explanation of why/how it's working). I'm not even sure the correct way to run it. 

Thank you!

**Comment by waylaidwanderer** [comment]
Simplified answer:
Jailbreak mode works by managing the conversation manually in the client, similar to ChatGPTClient, creating a new conversation every time and sending the chat log as context.
As for the Sydney persona, it's just inserting those instructions as part of the chat log.
